lib/existable.rb

```diff
@@ -1,5 +1,4 @@
-require 'helper'
-require 'findable'
+require_relative 'helper'
 
```

Comment:
Does this need require_relative 'findable'? Only difference I noticed!

Reply:
it shouldnt anymore because i had to rewrite a method anyway :)